tutorials/tmva/keras/ClassificationKeras.py

```diff
@@ -33,34 +32,38 @@
 dataloader.AddSignalTree(signal, 1.0)
 dataloader.AddBackgroundTree(background, 1.0)
 dataloader.PrepareTrainingAndTestTree(TCut(''),
-        'nTrain_Signal=4000:nTrain_Background=4000:SplitMode=Random:NormMode=NumEvents:!V')
+                                      'nTrain_Signal=4000:nTrain_Background=4000:SplitMode=Random:NormMode=NumEvents:!V')
 
 # Generate model
 
 # Define initialization
+
+
 def normal(shape, name=None):
     return initializations.normal(shape, scale=0.05, name=name)
 
+
 # Define model
 model = Sequential()
-model.add(Dense(64, init=normal, activation='relu', W_regularizer=l2(1e-5), input_dim=4))
+model.add(Dense(64, activation='relu', W_regularizer=l2(1e-5), input_dim=4))
 #model.add(Dense(32, init=normal, activation='relu', W_regularizer=l2(1e-5)))
-model.add(Dense(2, init=normal, activation='softmax'))
+model.add(Dense(2, activation='softmax'))
 
 # Set loss and optimizer
-model.compile(loss='categorical_crossentropy', optimizer=SGD(lr=0.01), metrics=['accuracy',])
+model.compile(loss='categorical_crossentropy',
+              optimizer=SGD(lr=0.01), metrics=['accuracy', ])
 
 # Store model to file
 model.save('model.h5')
 model.summary()
 
 # Book methods
 factory.BookMethod(dataloader, TMVA.Types.kFisher, 'Fisher',
-        '!H:!V:Fisher:VarTransform=D,G')
+                   '!H:!V:Fisher:VarTransform=D,G')
 factory.BookMethod(dataloader, TMVA.Types.kPyKeras, 'PyKeras',
-        'H:!V:VarTransform=D,G:FilenameModel=model.h5:NumEpochs=20:BatchSize=32')
+                   'H:!V:VarTransform=D,G:FilenameModel=model.h5:NumEpochs=20:BatchSize=32:Tensorboard=./logs')
```

Comment:
I would not set this as default in the tutorial's example. I assume that it fails if tensorboard is not installed and this would confuse most people.